runtime/src/snapshot_utils.rs

```diff
@@ -73,6 +72,15 @@ const MAX_SNAPSHOT_VERSION_FILE_SIZE: u64 = 8; // byte
 //         Snapshots created with versions <2.0.0 will fastboot to version 2.0.0
 const SNAPSHOT_FASTBOOT_VERSION: Version = Version::new(2, 0, 0);
 
+/// A package created from a snapshot request, containing information required to serialize the bank
+/// snapshot
+pub struct BankSnapshotPackage {
+    pub bank_fields: BankFieldsToSerialize,
+    pub bank_hash_stats: BankHashStats,
+    pub slot_deltas: Vec<BankSlotDelta>,
```

Comment:
I don't like the name either, but can we go back to having `status_cache` in the name? I at least feel that's more explanatory.


```suggestion
    pub status_cache_slot_deltas: Vec<BankSlotDelta>,
```

Reply:
Yeah, let's stick with `status_cache_slot_deltas`, since (1) that's what SnapshotPackage used. And (2), because it's the slot deltas *of the status cache*.